#!/usr/bin/env python

PACKAGE = "prj_drone"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("take_off",   bool_t,   0, "A Boolean parameter",  False)

gen.add("land",   bool_t,   0, "A Boolean parameter",  False)

gen.add("cambio_camara",   bool_t,   0, "A Boolean parameter",  False)

gen.add("velocidadX", double_t, 0, "A double parameter",    0, -0.2,0.2)
gen.add("velocidadY", double_t, 0, "A double parameter",    0, -0.2,0.2)
gen.add("velocidadZ", double_t, 0, "A double parameter",    0, -0.2,0.2)

gen.add("angularX", double_t, 0, "A double parameter",    0, -0.2,0.2)
gen.add("angularY", double_t, 0, "A double parameter",    0, -0.2,0.2)
gen.add("angularZ", double_t, 0, "A double parameter",    0, -0.2,0.2)

gen.add("z_desitjada", double_t, 0, "A double parameter",    1, 0,   3)
gen.add("x_desitjada", double_t, 0, "A double parameter",    0, -1,   1)
gen.add("y_desitjada", double_t, 0, "A double parameter",    0, -1,   1)



exit(gen.generate(PACKAGE, "prj_drone", "dynamic_params"))

